src/components/Asset/AssetContent/EditHistory.tsx

```diff
@@ -10,20 +10,38 @@ import styles from './EditHistory.module.css'
 const getReceipts = gql`
   query ReceiptData($address: ID!) {
     nftUpdates(
-      where: { id: $address }
+      where: { nft: $address }
       orderBy: timestamp
       orderDirection: desc
     ) {
       id
+      nft {
+        address
+      }
       tx
       timestamp
+      type
     }
   }
 `
 
 export default function EditHistory(): ReactElement {
   const { ddo } = useAsset()
 
+  function getUpdateType(type: string): string {
+    switch (type) {
+      case 'METADATA_CREATED':
+        return 'published'
+      case 'METADATA_UPDATED':
+        return 'updated'
+      case 'STATE_UPDATED':
+        return 'state updated'
+      case 'TOKENURI_UPDATED':
+        return 'token uri updated'
```

Comment:
not sure if people would properly understand this @kremalicious any ideas how the label should look ? `nft metadata updated` and for `METADATA_UPDATED` have `asset metadata updated` ?

Reply:
yup, think "NFT metadata updated" for this case is better, for asset updated we can leave as is with "updated"